### DISCUSSION TITLE
Hi can you explain orbitDB storage model

### DISCUSSION THREAD

**Discussion by surapuramakhil** [orbitdb/orbit-db#1208] [Q&A] [answered]
Assume there is a network of 5 p2p orbitDB instances

does orbitDB provide control on replica factor?

is data will always reside in all orbitDB instances - (means each create, update, delete operations are shared to peers) and each peers has a copy of entire data

is the data distributed across network?

**Comment by tabcat** [comment, marked as answer]
> does orbitDB provide control on replica factor

No, but a similar database [Peerbit](https://github.com/dao-xyz/peerbit) does.

> is data will always reside in all orbitDB instances

Eventually

**Reply by surapuramakhil** [reply]
So orbit is p2p database only. @tabcat any knowledge on distributed p2p db?
